### PR TITLE
EPA AQI for county and CSBA

### PR DESCRIPTION
**Pull request by n-h-diaz**
/bigstore/resolved_mcf/template_mcf_imports/epa/airdata/nodes.mcf-?????-of-?????,/bigstore/resolved_mcf/template_mcf_imports/epa/airdata/EPA_AQI.mcf

https://dashboards.corp.google.com/_51d0e7ea_3666_4fe8_82a6_acbf24d2585f?p=CNSPATH:%2Fcns%2Fjv-d%2Fhome%2Fdatcom%2Fimport_server%2Fdev%2Fvalidate_EPAAQIStats%2FValidationProto@100

### CODE REVIEW

File: scripts/epa/airdata/air_quality_aggregate.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+
+def write_csv(csv_file_path, reader):
+    with open(csv_file_path, 'a', newline='') as f_out:
+        writer = csv.DictWriter(f_out,
+                                fieldnames=CSV_COLUMNS,
+                                lineterminator='\n')
+        for observation in reader:
+            place = get_place(observation)
+            if place == None:
```
Comment (pradh) [inline scripts/epa/airdata/air_quality_aggregate.py:74]: nit: not place

File: scripts/epa/airdata/README.md [hunk 2]
```diff
@@ -24,19 +26,27 @@ and relies on the following StatisticalVariables:
 - Mean_Concentration_AirPollutant_PM10
 - Max_Concentration_AirPollutant_PM10
 - AirQualityIndex_AirPollutant_PM10
+- AirQualityIndex_AirPollutant (only for county/CSBA)
 
 ## Notes on the Data
-In the cleaned CSV, observations are provided on the site monitor level. For simplicity in the place mapping, we select a single monitor per site and pollutatn and provide the monitor id (POC) as airQualitySiteMonitor in the observation. So, a given observation is distinguished by observationDate, observationAbout (AirQualitySite), and measurementMethod (pollutant standard).
+Pollutant-specific metrics are provided on the site monitor level. For simplicity in the place mapping, we select a single monitor per site and pollutant and provide the monitor id (POC) as airQualitySiteMonitor in the observation. So, a given observation is distinguished by observationDate, observationAbout (AirQualitySite), and measurementMethod (pollutant standard).
+
+County/CBSA level AQI metrics include the defining site and pollutant, which are included in the StatVarObservation.
```
Comment (pradh) [inline scripts/epa/airdata/README.md:34]: I guess I'm confused what a "defining pollutant" is and why it is not in the StatVar.

File: scripts/epa/airdata/test_data/test_aggregate_import_data.csv [hunk 3]
```diff
@@ -0,0 +1,11 @@
+"State Name","county Name","State Code","County Code","Date","AQI","Category","Defining Parameter","Defining Site","Number of Sites Reporting"
```
Comment (pradh) [inline scripts/epa/airdata/test_data/test_aggregate_import_data.csv:1]: Have an test with CBSA code?

File: scripts/epa/airdata/README.md (hunk 2 above)
Comment (n-h-diaz) [reply scripts/epa/airdata/README.md:34]: this refers to the pollutant that is used to get the aggregate aqi measurement (I believe it's just the max out of all of the individual pollutant aqis)

File: scripts/epa/airdata/air_quality_aggregate.py (hunk 1 above)
Comment (n-h-diaz) [reply scripts/epa/airdata/air_quality_aggregate.py]: done

File: scripts/epa/airdata/test_data/test_aggregate_import_data.csv (hunk 3 above)
Comment (n-h-diaz) [reply scripts/epa/airdata/test_data/test_aggregate_import_data.csv]: added -- this was a good idea :) it helped catch a bug in my code - I'll rerun the import before manifesting
thanks!!

**Comment by n-h-diaz** [comment]
thanks for the review! :)

**Review by pradh — APPROVED** [review]
Thanks for the test!

**Pull request merged**